pandas/core/frame.py

```diff
@@ -3250,48 +3250,34 @@ def assign(self, **kwargs):
 
         Examples
         --------
-        >>> df = pd.DataFrame({'A': range(1, 11), 'B': np.random.randn(10)})
+        >>> df = pd.DataFrame({'temp_c': (17.0, 25.0)},
+                                index=['Portland', 'Berkeley'])
```

Comment:
This is missing `...` at the same level of the `>>>`, and the indentation is not correct